crates/driver/Cargo.toml

```diff
@@ -15,26 +15,34 @@ name = "driver"
 path = "src/main.rs"
 
 [dependencies]
+num = "0.4"
```

Comment:
I noticed at first that these don't use `{ workspace = true }` dependencies (which I personally like).

At first, I was going to suggest to change them back, but I realize that this is probably intentional as we intend to move it into a separate repo. Is this correct?